ui/src/app/pages/analysis/notebook-list.tsx

```diff
@@ -113,7 +172,17 @@ export const NotebookList = withCurrentWorkspace()(class extends React.Component
           </CardButton>
         </TooltipTrigger>
         <div style={{display: 'flex', flexWrap: 'wrap', justifyContent: 'flex-start' }}>
-          {notebookList.map((notebook, index) => {
+          {cloneLoadingNotebookMsg && <CardButton
+              disabled={cloneLoadingNotebookMsg}
+              style={styles.cloneNotebookCard}
+              type='small'
+              onClick={() => {}}>
+            <FlexColumn style={styles.cloneNotebookMsg}>
+              <div><FontAwesomeIcon icon={faClock} size="2x"></FontAwesomeIcon></div>
+              <div>Copying 1 or more notebooks from another workspace. May take a few minutes.</div>
```

Comment:
I prefer "This may take a few minutes" here

Reply:
Fixed